README.adoc

```diff
@@ -122,16 +93,18 @@ Now, navigate back to the `start` directory to begin.
 
 
 // =================================================================================================
-// Enabling fault tolerance in the application
+// Enabling fault tolerance
 // =================================================================================================
 
-== Enabling fault tolerance in the application
+== Enabling fault tolerance
 
 The MicroProfile Fault Tolerance API is added as a dependency to your `pom.xml` file. Look for
-the dependency with the `org.eclipse.microprofile.fault-tolerance` group ID. Use this feature
-to build fault-tolerant microservices with the MicroProfile Fault Tolerance API.
+the dependency with the `org.eclipse.microprofile.fault-tolerance` group ID.
+Add this dependency to use MicroProfile Fault Tolerance API in your code to use the
+fault tolerance policies in your microservices.
```

Comment:
Consider "to build fault tolerant microservices" for "to use the fault tolerance policies in your microservices"?

Reply:
This is already been changed, right? Looks like you commented on the red highlight (or old) text.

Reply:
Sorry if it wasn't clear.  Let me suggest this:  Add this dependency to use MicroProfile Fault Tolerance API in your code to make your microservices tolerate failures.